docs/src/_guide/troubleshooting.md

```diff
@@ -30,7 +30,17 @@ If you have a question about Meltano, are having trouble getting it to work, or
     </div>
 </div>
 
-# Common Troubleshooting Issues
+# Common Issues
+
+*Problem: "Why do **incremental runs** produce duplicate data?"* 
+
+View this [Slack discussion](https://meltano.slack.com/archives/C013EKWA2Q1/p1659024288228009). Tl;dr: Singer takes an "at least once" approach to replication, so if you're encounting this, it might be intended behavior.
```

Comment:
Users who are on this page are probably often not having a good time, and so may take this casual tone (e.g. using "tl;dr:") poorly. They may feel as if they and their problems are not being taken seriously.

Reply:
@WillDaSilva I was able to rewrite it w/o the TL;DR.